.gitignore

```diff
@@ -68,4 +68,4 @@ static
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
 
-.env
```

Comment:
This edit was from an extension's file not being omitted as expected. I removed it from the project folder entirely and restored the .gitignore to it's previous state.